lms/djangoapps/verify_student/views.py

```diff
@@ -1190,9 +1210,15 @@ def post(self, request, course_id, checkpoint_name, location):
                 'location': location
             }
             return render_to_response("verify_student/incourse_reverify.html", context)
+
         init_verification = SoftwareSecurePhotoVerification.get_initial_verification(user)
         if not init_verification:
-            log.error("Could not submit verification attempt for user %s", request.user.id)
+            log.info(
+                u"User %s does not have an initial verification, so "
+                u"he/she will be redirected to the \"verify later\" flow "
+                u"for the course %s.",
+                user.id, course_key
+            )
```

Comment:
optional: same long-ish log message occurring twice (and appears the surrounding logic is also similar), maybe can be factored out.